modules/programs/sm64ex.nix

```
@@ -0,0 +1,126 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.sm64ex;
+
+  # This is required for tests, we cannot overwrite the dummy package.
+  package = if ((cfg.region == null) && (cfg.baserom == null)
+    && (cfg.extraCompileFlags == null)) then
+    cfg.package
+
```

Can remove the unnecessary parentheses:
```suggestion
  package = if cfg.region == null && cfg.baserom == null
    && cfg.extraCompileFlags == null then
    cfg.package
```

---

The `format` utility decided to change the indentation a bit, but I don't think that should be an issue.